package.json

```diff
@@ -59,13 +59,11 @@
   },
   "repository": "github.com/sablier-labs/v2-core",
   "scripts": {
+    "benchmark": "bun run build:optimized && FOUNDRY_PROFILE=benchmark forge test --mt testGas && bun run prettier:write",
     "build": "forge build",
     "build:optimized": "FOUNDRY_PROFILE=optimized forge build",
     "build:smt": "FOUNDRY_PROFILE=smt forge build",
     "clean": "rm -rf artifacts broadcast cache docs out out-optimized out-svg",
-    "gas:report": "forge test --gas-report --mp \"./test/integration/**/*.sol\" --nmt \"test(Fuzz)?_RevertWhen_\\w{1,}?\"",
```

Comment:
We shouldn't have removed this function. It can still be helpful despite having benchmarks.

Reply:
Me and @andreivladbrg had a conversation on it [here](https://github.com/sablier-labs/v2-core/pull/918#discussion_r1596073797). I agreed with him on the point that its not accurate. These are average values determined by the tests.

Reply:
OK, let's remove it then